Tests/Approval.Tests/ApiApproval.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using PublicApiGenerator;
+using VerifyTests;
+using VerifyXunit;
+using Xunit;
+
+namespace Approval.Tests
+{
+    [UsesVerify]
+    public class ApiApproval
+    {
+        [Theory]
+        [InlineData("net47")]
+        // TODO: [InlineData("netstandard2.0")] throws AssemblyResolutionException for Jonas
+        public Task ApproveApi(string frameworkVersion)
```

Comment:
⛏️ (nitpick) This is a particularly ugly block of code.

Reply:
Is there a concrete change you'd like or is this just an observation?

Reply:
Only if you have time, hence I marked it as a nitpick.

Reply:
Alright.
Then I'll defer the prettifying to whoever gets to it first